bin/fix_my_workspace

```diff
@@ -4,8 +4,17 @@ $VERBOSE = nil  # Don't care about world writable dir warnings and the like
 if $_fix_my_workspace_version_check
   $_fix_my_workspace_version = '0.5.0'
 else
-  require 'origen'
-
+  # if we're running this from a development environment (i.e., bin/ inside of origen repo) then
```

Comment:
Probably I wasn't clear, but the point is that if this change is introducing something that breaks the fix_my_workspace script, then we need a parallel update to the master version of that script to make it work with your update but which is also backwards compatible with existing Origen.

This is the master version here: https://github.com/Origen-SDK/origen_updater/blob/master/bin/fix_my_workspace

You should never make edits to this file within an application like this, since they will get overridden by updating the version of origen_updater.